R/tt_export.R

```diff
@@ -166,7 +166,7 @@ export_as_txt <- function(tt, file = NULL,
                           paginate = .need_pag(page_type, pg_width, pg_height, lpp, cpp),
                           cpp = NULL,
                           lpp = NULL,
-                          ..., page_break = "\\s\\n",
+                          ..., page_break = "\\f",
```

Comment:
I must have lost the discussion but why should we use these page break indicators? I mean how this was chosen?